src/platform/plugins/shared/dashboard/public/services/logger.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import type { Logger } from '@kbn/logging';
+
+export let logger: Logger;
+
+export const setLogger = (_logger: Logger) => {
```

Comment:
I wasn't sure if to have it separate like this or to add to existing `setKibanaServices`, I opted for a separate because `logger` isn't part of `CoreStart` but available on plugin initialize context. It also can be usefull before start phase. But lmk if I should change this and merge into `setKibanaServices`